src/components/Table/index.jsx

```diff
@@ -1,36 +1,44 @@
 import TableRow from "../TableRow/index.jsx";
+import JobDescriptionModal from "../JobDescriptionModal/index.jsx";
+import {useState} from "react";
 
 const Table = ({jobInfo}) => {
+    const [modalID, setModalID] = useState(1)
     return (
-        <table className='table table-dark table-striped ms-1 ms-md-0 table-sm'>
-            <thead>
-            <tr>
-                <th scope='col'>Job title / Company</th>
-                <th scope='col'>Type</th>
-                <th scope='col'>Salary</th>
-                <th scope='col'>Skills</th>
-            </tr>
-            </thead>
-            <tbody>
-            {
-                jobInfo.map((item) => {
-                        return (
-                            <TableRow
-                                key={item.id}
-                                jobTitle={item.job_title}
-                                company={item.company}
-                                contract={item.type}
-                                salary={item.salary}
-                                skills={item.skills}
-                                logo={item.logo}
-                            />
-                        )
-                    }
-                )
-            }
-            </tbody>
-        </table>
-    )
+        <div>
+            <table className='table table-dark table-striped ms-1 ms-md-0 table-sm'>
+                <thead>
+                <tr>
+                    <th scope='col'>Job title / Company</th>
+                    <th scope='col'>Type</th>
+                    <th scope='col'>Salary</th>
+                    <th scope='col'>Skills</th>
+                </tr>
+                </thead>
+                <tbody>
+                {
+                    jobInfo.map((item) => {
+                            return (
+                                <TableRow
+                                    key={item.id}
+                                    id={item.id}
+                                    details={setModalID}
```

Comment:
I don't like that this prop is called `details`, it doesn't really tell you what it does. I'm struggling to think of a better name, though! I think even just reusing `setModalID` would be better, but open to ideas.

Reply:
Refactored details -> jobIdSelector, id -> jobId. Feels more informative and specific.